index.html

```diff
@@ -58,6 +63,8 @@
       <section id='test' class='project four columns template offset-by-one column'>
       </section>
       <h1 id='totes' class=''></h1>
+      <h1 id='totalProjects' class='project'></h1>
```

Comment:
You're using too many \<h1\> elements. There are h1-h6 heading elements to specify importance and assist with semantics.